Tools/ardupilotwaf/boards.py

```diff
@@ -214,6 +233,9 @@ def configure_env(self, cfg, env):
             env.LIB += [
                 'winmm',
             ]
+            
+    def __init__(self):
+        self.with_uavcan = True
```

Comment:
Travis is (almost) always right. If there's an error we need to fix the source code to fix it, not just adding UAVCAN to the build.